csds-client/README.md

```diff
@@ -10,7 +10,7 @@ For now, this initial version of this CSDS client only supports GCP's [Traffic D
 # Running
 * run with `./bin/csds <flag>`, e.g. <br/><br/>
    * auto authentication mode
-   ```
+   ```bash
    ./bin/csds \
      -service_uri <uri> \
```

Comment:
maybe consider change this to csds-client?

Reply:
+1

Reply:
I've modified the Makefile to run `go install` to build the binary under `GOPATH/bin`. In this way, the binary will be renamed as `csds-client` and we can just run it with `csds-client <flag>`